README.md

```diff
@@ -1,2 +1,14 @@
 # ops-requests
-Requests about infrastructure operations
+
+Use this repo's issue tracker to direct requests to the infrastructure team (mainly @lgierth).
+
+Make sure that:
+
+- the request is specific
+- the request is actionable
+- the request is prioritizable
+  - e.g. has a specific timeframe
```

Comment:
Could you add a description to the README of what the labels(https://github.com/ipfs/ops-requests/labels) mean? Urgent is Urgent, but the others might not mean the same thing to everyone.